client/src/screens/schedules.screen.js

```diff
@@ -74,7 +75,15 @@ class Schedule extends Component {
   }
 
   render() {
-    const { id, name } = this.props.schedule;
+    const { id, name, details, startTime, endTime } = this.props.schedule;
+    let timeText = '';
+    if (startTime === 0 && endTime === distantFuture) {
+      timeText = 'Perpetual Schedule';
+    } else {
+      const startText = moment.unix(startTime).format('DD/MM/YY, HH:mm:ss');
+      const endText = moment.unix(endTime).format('DD/MM/YY, HH:mm:ss');
+      timeText = `${startText} - ${endText}`;
```

Comment:
this is much better than how you did it in `groups.screen`. Wanna make the other one look like this?

Reply:
Lol yea, much shame. very sorry.